prep-html-links/contact-us.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div>
+    <a href="file:///C:/Users/balta/repos/prep-0421-code-solutions/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
Here you want to use a relative path to the index file instead of a full absolute path (`../index.html`).